Daytime-Operations/Auxiliary-Telescope/AT-Daytime-Checkout/Daytime Checkout SAL Scripts.rst

```diff
@@ -0,0 +1,93 @@
+.. |author| replace:: *E. Dennihy*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _AT-Daytime-Checkout-Daytime-Checkout:
+
+############################
+Daytime Checkout SAL Scripts
+############################
+
+To ensure the Auxiliary Telesocpe is fully operational and ready for nighttime use, we have created a series of SAL Scripts that successively 
+checkout each of the major components, culminating in a full test of the system involving slewing to a target, tracking, and taking a test image
+with LATISS. The goal of these scripts is to test the functionality of each component within the current software environment,
+and alert the daytime staff to any operational issues before going on-sky. These tests are designed to be run during the 
+daytime with an Observing Specialist or other personnel available to verify their results. 
+
+The scripts are intended to be run in order, but offer some flexibility in what state to leave the Auxiliary Telescope (e.g. 
+ready to take calibrations, ready to observe at night, or simply parked and shutdown). They can also be run individually if a 
+single component is undergoing maintenence and requires a check of basic functionality. 
+
+None of the scripts require configuration. 
+
+.. Important::
+
+    The daytime checkout scripts involve Telescope, Dome, and Instrument component movement. You must announce that you are running
+    the checks on the #summit-announce slack channel before execution. 
```

Comment:
Maybe you can add a note that people should check that the telescope surrounding area is free. It usually is, but if daytime is ongoing, we never know.